skbio/io/format/tests/test_stockholm.py

```diff
@@ -376,5 +389,109 @@ def test_handles_missing_metadata_efficiently(self):
         self.assertIsNone(msa[0]._positional_metadata)
 
 
+class TestStockholmWriter(unittest.TestCase):
```

Comment:
Can you add these writer tests?

- [ ] error is raised if TabularMSA's index isn't unique (will produce an unreadable Stockholm file)
- [ ] error if duplicate GC or GR positional metadata columns present in TabularMSA (will produce an unreadable Stockholm file)
- [ ] test writing an empty MSA (no sequences)
- [ ] test writing an MSA without any metadata handles metadata efficiently (e.g., see the corresponding reader test we wrote)
- [ ] test writing an MSA with a non-string index and metadata key/values (should be converted to string)